lib/runnerman.js

```diff
@@ -1,18 +1,19 @@
 const { Collection, Event, Script } = require('postman-collection');
 const { run: newman } = require('newman');
+const { format } = require('util');
+const { v4: uuidv4 } = require('uuid');
 const { filterCollection } = require('./util/filterCollection');
 const { generateGlobals } = require('./util/generateGlobal');
-const { format } = require('util');
 const { createGlobalVariable } = require('./util/createGlobalVariable');
-const { v4: uuidv4 } = require('uuid');
 
-module.exports = async function (suite, collectionDefinition, environment, iterations) {
+
+module.exports = async function runnerman(suite, collectionDefinition, environment, iterationCount, reporters) {
     let guid = null;
```

Comment:
Esse guid aqui não tem uso. Dê preferência a sua geração em variável exclusiva dentro do escopo onde ela será utilizada.

Reply:
I changed this variable to a property in `Summary`.